src/main/java/de/slimecloud/slimeball/features/highlights/HighlightCommand.java

```diff
@@ -0,0 +1,53 @@
+package de.slimecloud.slimeball.features.highlights;
+
+import de.mineking.discordutils.DiscordUtils;
+import de.mineking.discordutils.commands.ApplicationCommand;
+import de.mineking.discordutils.commands.ApplicationCommandMethod;
+import de.mineking.discordutils.commands.Command;
+import de.mineking.discordutils.commands.Setup;
+import de.mineking.discordutils.commands.condition.Scope;
+import de.mineking.discordutils.commands.context.ICommandContext;
+import de.mineking.discordutils.commands.option.Option;
+import de.mineking.discordutils.list.ListManager;
+import de.slimecloud.slimeball.main.SlimeBot;
+import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent;
+import org.jetbrains.annotations.NotNull;
+
+@ApplicationCommand(name = "highlight", description = "Verwaltet Highlights", scope = Scope.GUILD_GLOBAL)
+public class HighlightCommand {
+	@Setup
+	public static void setup(@NotNull SlimeBot bot, @NotNull DiscordUtils<?> discordUtils, @NotNull ListManager<ICommandContext> list, @NotNull Command<ICommandContext> command) {
+		discordUtils.getJDA().addEventListener(new HighlightListener(bot));
+
+		command.addSubcommand(list.createCommand(state -> bot.getHighlights()).withDescription("Zeigt deine Highlights für diesen Server an"));
+	}
+
+	@ApplicationCommand(name = "add", description = "Füge ein Highlight auf diesem Server hinzu")
+	public static class HighlightAddCommand {
```

Comment:
Don't re-add when the highlight already exists and send a message for that

Reply:
The set method already checks whether the highlight already exists

Reply:
But you should also give corresponding feedback to the user. For example return a boolean in the set method indicating whether the highlight was added or already existed